templates/index.html

```diff
@@ -1,18 +1,66 @@
-{% extends "layout.html" %}
+{% extends 'layout.html'  %}
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+<div class="jumbotron">
+    <h1 style="color: blue" class="display-4">To-Do App</h1>
+    <p style="color: blue" class="lead">Just another to-do app.</p>
+</div>
+<div class="well text-container">
+    <div class="row">
+        <div class="col-md-12">
+            <h2 style="color: blue">Items</h2>
+            <ul class="list-group mb-4">
+                <h6>
+                    {% for todo in todos %}
+                    {{ todo.title }} {{todo.status}}
+                    <a href="/" class="btn btn-danger">Item to Delete</a>
```

Comment:
You'll want to link this to a delete endpoint (and probably move this to a form)